app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Kindly remove the `foreign_key: 'author_id'`. Remember that the `User` has_many `Posts`, therefore the `foreign_key` will be on the `Posts` Table. 👍